tools/pbxproj_generators/project_prefix/test/CalculateProjectPrefixTests.swift

```diff
@@ -0,0 +1,179 @@
+import CustomDump
+import Foundation
+import PBXProj
+import XCTest
+
+@testable import project_prefix
+
+class CalculateProjectPrefixTests: XCTestCase {
+    func test_noOrganizationName() {
+        // Arrange
+        
+        let compatibilityVersion = "AppCode 42.7.4"
+        let developmentRegion = "en-GB"
+        let organizationName: String? = nil
+        let projectDir = "/some/execution_root"
+        let workspace = "/Users/TimApple/StarBoard"
+
+        let expectedProjectPrefix = #"""
+		000000000000000000000001 /* Project object */ = {
+			isa = PBXProject;
```

Comment:
This uses tabs for the indent, which is expected in the output. You can't easily see that though, and I bet it can be a pain for development. Currently I opted for here and in `Generator.calculateProjectPrefix()` to directly use them, since `\t` is hard to read, and developing an indent system seemed like a lot of overhead when we will always know the exact indent we want.

I'm open to feedback though. Maybe we just need a comment pointing out that it uses tabs and it's intentional?

Reply:
I wouldn't over engineer it as well, IMO a comment with details should be enough